src/smartpeak/source/ui/Widget.cpp

```diff
@@ -123,16 +123,16 @@ namespace SmartPeak
 
     // headers
     const ImGuiTableFlags table_flags = ImGuiTableFlags_Resizable | ImGuiTableFlags_Hideable |
-      //ImGuiTableFlags_Sortable | 
-      ImGuiTableFlags_Borders | ImGuiTableFlags_Scroll | ImGuiTableFlags_ScrollFreezeTopRow | ImGuiTableFlags_ScrollFreezeLeftColumn;
```

Comment:
Are the flags for freezing the top row and freezing the left column no longer available or did the names change?

Reply:
They are no longer included as flags, but possible with a function call as per ocornut's comment : [BREAKING CHANGES (SEPT 2020)](https://github.com/ocornut/imgui/issues/2957#issuecomment-698317698)

Reply:
Thanks for clarifying this as well and sharing the link.

Hopefully, this information will come in handy for upgrading the old columns API to the tables API in the next round of PRs.